src/components/Router/index.js

```diff
@@ -2,11 +2,14 @@ import { h, Component } from 'preact'
 import { bindActionCreators } from 'redux'
 import { connect } from 'react-redux'
 import io from 'socket.io-client'
-
 import { componentsList } from './StepComponentMap'
 import { unboundActions } from '../../core'
 import StepsRouter from './StepsRouter'
 import { isDesktop } from '../utils'
+import createHistory from 'history/createBrowserHistory'
```

Comment:
Minor: We are trying to keep external dependencies at the top of the file, with a line between externals and  internal ones. Could you move it after `import io from 'socket.io-client'`?